cw-orch-daemon/src/sync/builder.rs

```diff
@@ -128,6 +130,13 @@ impl DaemonBuilder {
         self
     }
 
+    /// Load network from `$HOME/.cw-orchestrator/networks.json`
```

Comment:
Do we want to change this to toml before merging?

Reply:
Changed!